src/Stratis.Bitcoin.Tests/P2P/PeerConnectorTests.cs

```diff
@@ -49,12 +50,10 @@ public void PeerConnectorAddNode_FindPeerToConnectTo_Returns_AddNodePeers()
             peerAddressManager.AddPeer(networkAddressAddNode, IPAddress.Loopback);
             peerAddressManager.AddPeer(networkAddressDiscoverNode, IPAddress.Loopback);
 
-            var nodeSettings = new NodeSettings
-            {
-                ConnectionManager = new Configuration.Settings.ConnectionManagerSettings()
-            };
+            var nodeSettings = new NodeSettings();
```

Comment:
Dont we need to instantiate connectionSettings? (using the Load method?)

Reply:
Done.